target/linux/generic/pending-5.4/611-netfilter_match_bypass_default_table.patch

```diff
@@ -80,7 +80,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
 -	WARN_ON(!(table->valid_hooks & (1 << hook)));
 -	local_bh_disable();
  	addend = xt_write_recseq_begin();
--	private = rcu_access_pointer(table->private);
+-	private = READ_ONCE(table->private); /* Address dependency. */
```

Comment:
Another appearance of `private = rcu_access_pointer(table->private);` is at line 51, should it be replaced too?

Reply:
I do not know.  I emailed the patch's author requesting a review.

Reply:
In this update of kernel 5.10 also the other place is changed: 
https://git.openwrt.org/?p=openwrt/staging/ldir.git;a=commitdiff;h=9473df3c6eaeaf98bbcebe1d0ec7333561d7ef56

Reply:
The patch itself essentially seems to move code, so I'd say "yes", too.

Reply:
The relevant kernel patch is a revert. So I've looked up our relevant commit introducing the reverted kernel patch:
https://github.com/openwrt/openwrt/commit/9b1b89229f0efef43a738f91875e0fd9fcc42277

This touches both locations, so "revert" both now. I will take care and probably merge the PR afterwards.